config/environments/production.rb

```diff
@@ -7,6 +7,7 @@
 
   # Code is not reloaded between requests.
   config.cache_classes = true
+  config.hosts << 'platform-console-api.vfs.va.gov'
```

Comment:
This shouldn’t be needed in production. I think the issue is `RAILS_ENV=production` needs to be passed when building the docker image

Reply:
Ah ok maybe I need to set an ENV var in the deployment.yml